examples/add-receiver-creator/README.md

```diff
@@ -0,0 +1,19 @@
+# Example of chart configuration
+
+## Add Receiver Creator
+This example shows how to add a receiver creator to the OTel Collector configuration
+[Receiver Creator](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/receivercreator).
+In this example we will configure it to observe kubernetes pods and in case there is a pod
+using port 5432 the collector will dynamically create a smartagent/postgresql receiver to monitor it.
```

Comment:
I assume it's an already existing example here, but why are we using a smartagent receiver here? The collector has a fully functioning postgresql receiver so we may as well use it. If it's more work than a couple line changes feel free to ignore.